Kickstarter-iOS/Views/ProjectStatesContainerView.swift

```diff
@@ -0,0 +1,88 @@
+import Library
+import KsApi
+import Prelude
+import UIKit
+
+class ProjectStatesContainerView: UIView {
+  fileprivate let vm: ProjectStatesContainerViewViewModelType = ProjectStatesContainerViewViewModel()
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var labelStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var backerLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var label: UILabel = { UILabel(frame: .zero) }()
+  private lazy var button: UIButton = {
+     return MultiLineButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    NSLayoutConstraint.activate(
+      [self.button.heightAnchor.constraint(greaterThanOrEqualToConstant: Styles.minTouchSize.height)]
+    )
+
+    _ = ([self.backerLabel, self.label], self.labelStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.labelStackView, self.button], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.button.rac.title = self.vm.outputs.buttonTitleText
+    self.button.rac.backgroundColor = self.vm.outputs.buttonBackgroundColor
+    self.labelStackView.rac.hidden = self.vm.outputs.stackViewIsHidden
+    self.label.rac.text = self.vm.outputs.rewardTitle
+  }
+
+  func configureWith(project: Project, user: User) {
+    _ = self.button
```

Comment:
Should all of these styling things go under the `bindStyles` function instead of `configureWith`?